src/com/company/Controller/Command/Connect.java

```diff
@@ -0,0 +1,45 @@
+package com.company.Controller.Command;
+
+import com.company.model.DatabaseManager;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+/**
+ * Created by yulia on 06.11.16.
+ */
+public class Connect extends AbstractCommand {
+
+    public Connect() {}
+    public Connect(DatabaseManager databaseManager) {
+        super(databaseManager);
+    }
+
+    @Override
+    public boolean shouldProcess(String command) {
+        return "connect".equals(command);
+    }
+
+    @Override
+    public void process(String command) throws SQLException, ClassNotFoundException {
+        boolean result = false;
```

Comment:
Considering written below, variable can be renamed to represent state of connection. Like 
**connectionSuccessful**